WIP-CHANGELOG.md

```diff
@@ -4,16 +4,69 @@
 
 # Updated
 
- - updated falco helm chart to version 1.16.0, this upgrades falco to version 0.30.0
+- updated falco helm chart to version 1.16.0, this upgrades falco to version 0.30.0
+- kube-prometheus-stack to v19.2.2 [#685](https://github.com/elastisys/compliantkubernetes-apps/pull/685)
```

Comment:
We had a short discussion offline regarding this changelog entry and we concluded that it should be reduced, to only include that of which is important to our product. Please keep the link to the full changelogs though.

Reply:
done please check.